### PR TITLE
INT-80: Handle edit loading error - when transitiong from article route to edit route

### PR DESCRIPTION
**Pull request by inez**

### CODE REVIEW

**Comment by inez** [comment]
@rogatty

File: front/scripts/main/routes/EditRoute.ts [hunk 1]
```diff
@@ -9,6 +9,10 @@ App.EditRoute = Em.Route.extend({
     },
 
     actions: {
+        error: function(error, transition) {
```
Comment (rogatty) [inline front/scripts/main/routes/EditRoute.ts]: Add missing types.

File: front/locales/en/main.json [hunk 2]
```diff
@@ -62,6 +62,8 @@
 		"cancel-label": "Cancel",
 		"edit-page-heading": "Editing:",
 		"edit-page-publish": "Publish",
-		"edit-success": "Successfully edited page __pageTitle__!"
+		"edit-success": "Successfully edited page __pageTitle__!",
+		"edit-load-error": "Something went wrong."
+
```
Comment (rogatty) [inline front/locales/en/main.json]: No need for this `\n`

File: front/scripts/main/routes/EditRoute.ts (hunk 1 above)
Comment (kenkouot) [inline front/scripts/main/routes/EditRoute.ts]: We do spaces after the `function` keyword. ORRRRRR you can use es6 object method shorthand syntax, eg: `error () {`

**Comment by Grunny** [comment]
LGTM.

File: front/locales/en/main.json [hunk 3]
```diff
@@ -62,6 +62,7 @@
 		"cancel-label": "Cancel",
 		"edit-page-heading": "Editing:",
 		"edit-page-publish": "Publish",
-		"edit-success": "Successfully edited page __pageTitle__!"
+		"edit-success": "Successfully edited page __pageTitle__!",
+		"edit-load-error": "Something went wrong."
```
Comment (Grunny) [inline front/locales/en/main.json]: Updated this error message to what Yoko specified in the ticket in https://github.com/Wikia/mercury/pull/834.

**Pull request merged**